static/scss/footer.scss

```diff
@@ -1,13 +1,79 @@
 // sass-lint:disable mixins-before-declarations
 
 .footer {
+
   background: $footer-bg;
-  border-top: 5px solid $footer-border;
-  padding: 30px 0 20px;
   color: white;
   font-weight: 500;
 
   .footer-top {
+    background: $footer-bg;
```

Comment:
Based on https://invis.io/QGRVI8DV4WM#/361801847_Version_2 the background color for the top section should be changed to `#F5F5F5`

Reply:
there is some difference in help & contact icons. please provide the `.psd` or the icons.

Reply:
Can you extract them from InVision, using the inspect function? I sent you an invite in case that's necessary. 

https://projects.invisionapp.com/d/main#/projects/prototypes/17454540

Reply:
@pdpinch I tried but did not find the icons inside assets.